deepmd/utils/model_stat.py

```
@@ -16,11 +16,35 @@
+    if get_stat_batch is not None:
+        return get_stat_batch(sys_idx)
+    return data.get_batch(sys_idx=sys_idx)
+
+
 def _make_all_stat_ref(data: Any, nbatches: int) -> dict[str, list[Any]]:
     all_stat = defaultdict(list)
-    for ii in range(data.get_nsystems()):
-        for jj in range(nbatches):
-            stat_data = data.get_batch(sys_idx=ii)
+    for ii in range(_get_stat_nsystems(data)):
+        for jj in range(_get_stat_numb_batches(data, ii, nbatches)):
```

Ruff (0.15.21)

[warning] Loop control variable `jj` not used within loop body

Rename unused `jj` to `_jj`

(B007)